annotationPipeline/src/main/resources/application.properties.EXAMPLE

```diff
@@ -0,0 +1,6 @@
+spring.batch.job.enabled=false
+genomenexus.base_url=
+genomenexus.hgvs=variant_annotation/hgvs/
+genomenexus.isoform_override=variant_annotation/isoform_override/
```

Comment:
I think it's probably better as a runtime argument. I did it this way just because the current way we set the isoforms annotation is through the properties file.